### PR TITLE
Add External Partners

### PR DESCRIPTION
**Pull request by Hgjnnf**
- Change Club page to Partner page
- Add Quiver Games and Google to Partner page
- Add Partner page link to Nav Bar
image
image

### CODE REVIEW

File: content/about/partners.md [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
+title: Partners
 layout: single
 # Card images should all be 348x196
 ---
 
+## Sponsors
+
+<div class="row">
+  <div class="col-lg-6 col-xl-4 mb-4 d-flex align-items-stretch">
+	{{< club title="Google" href="" src="/files/google_logo.png" id="google">}}
```
Comment (kewbish) [inline content/about/partners.md:11]: Could you `href` to the Google uni recruiting page (or something else @wgervasio?)

**Review by kewbish — COMMENTED** [review]
Thank you!

Could we create a new shortcode for partner companies? It doesn't make sense to have the popover with only one link. We should also add a little padding to the partner images — the club logos already had padding in the image, but the company logos don't.

File: content/about/partners.md (hunk 1 above)
Comment (wgervasio) [reply content/about/partners.md:11]: i think href to the recruiting page is good @kewbish

**Comment by wgervasio** [comment]
modified tech career north's smaller logo and edited their description. The logo sizing is a little funky but looks good otherwise! Thank you!!

image

**Review by kewbish — APPROVED** [review]
Thank you :pray::pray:

**Pull request merged**